modules/navier_stokes/src/fvkernels/INSFVMomentumDiffusionComplete.C

```diff
@@ -0,0 +1,204 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "INSFVMomentumDiffusionComplete.h"
+#include "INSFVRhieChowInterpolator.h"
+#include "NS.h"
+#include "SystemBase.h"
+#include "RelationshipManager.h"
+#include "Factory.h"
+
+registerMooseObject("NavierStokesApp", INSFVMomentumDiffusionComplete);
+
+InputParameters
+INSFVMomentumDiffusionComplete::validParams()
```

Comment:
shouldn't this be: 
`nabla(mu*(grad(u) + grad(phi)^T-2/3div(u)*I))`? I suppose the div(u) part will only show up if we do weakly compressible